app/views/layouts/posts/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Here is a list of posts</h1>
```

Comment:
- [ ] Please make sure to place the `posts` views the entire `posts` folder inside the views directory as you did for `users` views. Currently, It's inside the `/layouts` folder.

![image](https://user-images.githubusercontent.com/83803414/236914586-b7a5400d-28b1-4c51-bbb0-346588376164.png)